airbyte-integrations/connectors/source-redshift/src/main/java/io/airbyte/integrations/source/redshift/RedshiftSource.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.source.redshift;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.base.Source;
+import io.airbyte.integrations.source.jdbc.AbstractJdbcSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RedshiftSource extends AbstractJdbcSource implements Source {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(RedshiftSource.class);
+
+  // todo (cgardens) - clean up passing the dialect as null versus explicitly adding the case to the
+  // constructor.
+  public RedshiftSource() {
+    super("com.amazon.redshift.jdbc.Driver", null);
```

Comment:
does this run right now?  i know by default the `AbstractJdbcSource` uses jooq, so if no dialect is set, i thought things failed. I expected you would either need to set dialect to postgres here OR would need to implement `getCurrentDatabaseName` and `getTables` like `MssqlSource`.

Reply:
just double checking, things run okay with this configured this way? i feel like jooq shouldn't work without a dialect selected, but if it does great i guess! it would be good to know what dialect it's selecting though.